app.py

```diff
@@ -4,6 +4,10 @@
 from flask_expects_json import expects_json
 import datetime
 
+from rq_function import add, count_lines, poppunk_assign
+from redis import Redis
+from rq import Queue
```

Comment:
I don't know what python best practice is re imports. Probably good to ask Alex to have a look at this too.

Reply:
These lines are "correct" as per PEP8: https://www.python.org/dev/peps/pep-0008/#imports

Above, lines 1 - 3 shoud be inlined to just 
`from flask import Flask, request, jsonify`